src/console_reporter.cc

```diff
@@ -25,10 +25,13 @@
 #include "colorprint.h"
 #include "string_util.h"
 #include "walltime.h"
+#include "benchmark_mpi.h"
 
 namespace benchmark {
 
 bool ConsoleReporter::ReportContext(const Context& context) {
+  if(not mpi_is_world_root())
```

Comment:
This PR has many style issues, I'll let @dominichamon comment on them, but this statement is missing a space before `(` and the code in this project uses `!` not `not`.